platform-tests/src/test/java/org/nd4j/imports/tfgraphs/TFGraphTestAllHelper.java

```diff
@@ -151,20 +155,30 @@ protected static void checkOnlyOutput(Map<String, INDArray> inputs, Map<String,
             outputsToCheck.add(s);
         }
 
-        Pair<SameDiff,Map<String,INDArray>> p = getGraphAfterExec(baseDir, modelFilename, modelName, inputs, execType, loader, null, outputsToCheck, printArraysDebugging);
-        SameDiff graph = p.getFirst();
-        Map<String,INDArray> sameDiffPredictions = p.getSecond();
+
 //        SameDiff graph = graphLoaderFunction.apply(new ClassPathResource(baseDir + "/" + modelName + "/" + modelFilename).getFile(), modelName);
         //Collect coverage info about ops
-       /* TensorflowFrameworkImporter tensorflowFrameworkImporter = new TensorflowFrameworkImporter();
+        TensorflowFrameworkImporter tensorflowFrameworkImporter = new TensorflowFrameworkImporter();
         File oldModel = new ClassPathResource(baseDir + "/" + modelName + "/" + modelFilename).getFile();
-        GraphDef g  = GraphDef.parseFrom(IOUtils.toByteArray(oldModel.toURI()));
-        TensorflowIRGraph tensorflowIRGraph = new TensorflowIRGraph(g,tensorflowFrameworkImporter.getOpDefList(),tensorflowFrameworkImporter.getRegistry());
-        TensorflowIRGraphRunner tensorflowIRGraphRunner = new TensorflowIRGraphRunner(tensorflowIRGraph,p.getFirst().inputs(),outputsToCheck.stream().collect(Collectors.toList()));
-        Map<String,INDArray> outputs = tensorflowIRGraphRunner.run(inputs);
-        SameDiff oldForComparison = TFGraphMapper.importGraph(oldModel);
-        Map<String,INDArray> oldOutputs = oldForComparison.outputAll(inputs);
-        */
+        Map<String,INDArray> oldOutputs =  null;
+        Map<String,INDArray> outputs = null;
+        SameDiff oldForComparison = null;
+/*
+        if(modelName.equals("math_mul_order")) {
+            GraphDef g  = GraphDef.parseFrom(IOUtils.toByteArray(oldModel.toURI()));
+            oldForComparison = TFGraphMapper.importGraph(oldModel);
+            oldOutputs = oldForComparison.outputAll(inputs);
+            TensorflowIRGraph tensorflowIRGraph = new TensorflowIRGraph(g,tensorflowFrameworkImporter.getOpDefList(),tensorflowFrameworkImporter.getRegistry());
+            TensorflowIRGraphRunner tensorflowIRGraphRunner = new TensorflowIRGraphRunner(tensorflowIRGraph,new ArrayList<>(inputs.keySet()),new ArrayList<>(oldOutputs.keySet()));
+            //outputs = tensorflowIRGraphRunner.run(inputs);
+            System.out.println();
+        }
```

Comment:
Lots of commented out code here and an empty println. 

This doesn't look merge ready.